ambari-server/src/main/java/org/apache/ambari/server/stack/StackManager.java

```diff
@@ -100,7 +99,7 @@
   /**
    * Map of stack id to stack info
    */
-  protected Map<String, StackInfo> stackMap = new HashMap<>();
+  protected TreeMap<String, StackInfo> stackMap = new TreeMap<>();
```

Comment:
Can it be declared `NavigableMap` instead of the implementation?

Reply:
Sure - I just got sick of my stack entries always being out of order on new installs - made it hard to diagnose stack-related issues. I'll change it.